docs/providers/aws/guide/variables.md

```diff
@@ -292,7 +292,7 @@ service: new-service
 provider:
   name: aws
   runtime: nodejs6.10
-  variableSyntax: "\\${{([ :a-zA-Z0-9._,\\-\\/\\(\\)]+?)}}" # notice the double quotes for yaml to ignore the escape characters!
+  variableSyntax: "\\${([ ~:a-zA-Z0-9._\'",\\-\\/\\(\\)]+?)}" # notice the double quotes for yaml to ignore the escape characters!
```

Comment:
The outside quoting is different here... may need to escape the inner double quote.